src/services/user.service.js

```diff
@@ -0,0 +1,36 @@
+const { User } = require('../models/User.model');
+const { generateRandomId } = require('../Random');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const getById = async (id) => {
+  return User.findByPk(+id);
```

Comment:
Consider adding error handling for the case where a user with the given ID does not exist. This will help in providing more informative responses to the client.